tests/components/slack/test_notify.py

```diff
@@ -0,0 +1,66 @@
+"""Test slack notifications."""
+from unittest.mock import Mock
+
+from homeassistant.components.slack.notify import SlackNotificationService
+
+from tests.async_mock import AsyncMock
+
+
+async def test_message_includes_default_emoji():
+    """Tests that overriding the default icon emoji when sending a message works."""
```

Comment:
Please update this docstring to correctly explain this test case.

Reply:
Updated as requested in 00f5feb.